analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp7.cs

```diff
@@ -288,12 +301,12 @@ public void MutedCase()
         {
             var tmp = 0;
             var flag = true;
-            while (flag) // FN, all "flag" issues are muted by presence of tuple assignment
+            while (flag)            // Noncompliant
             {
                 tmp = 0;
             }
 
-            while (flag) // Compliant, muted by presence of tuple assignment
+            while (flag)            // Compliant, muted by presence of tuple assignment
```

Comment:
```suggestion
            while (flag)            // Compliant
```
My understanding is that the tuple is not an issue anymore.